docs/modules/agents/tools/getting_started.md

```diff
@@ -160,3 +160,9 @@ Below is a list of all supported tools and relevant information:
 - Notes: A connection to the OpenWeatherMap API (https://api.openweathermap.org), specifically the `/data/2.5/weather` endpoint.
 - Requires LLM: No
 - Extra Parameters: `openweathermap_api_key` (your API key to access this endpoint)
+
+**sleep**
```

Comment:
Let's add to `langchain/agents/load_tools.py` for this one!

Reply:
Thanks, done.